### PR TITLE
test: add tests where coverage was lacking

### PR DESCRIPTION
**Pull request by ArneD**

### CODE REVIEW

File: test/StreetNameRegistry.Tests/AggregateTests/WhenCorrectingNameToClearedMunicipality/GivenMunicipalityWasAlreadyNamed.cs [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+            _municipalityId = Fixture.Create<MunicipalityId>();
+            _streamId = Fixture.Create<MunicipalityStreamId>();
+        }
+
+        [Theory]
+        [InlineData(Language.Dutch)]
+        [InlineData(Language.French)]
+        [InlineData(Language.English)]
+        [InlineData(Language.German)]
+        public void ThenNamedAgain(Language language)
```
Comment (jvandaal) [inline test/StreetNameRegistry.Tests/AggregateTests/WhenCorrectingNameToClearedMunicipality/GivenMunicipalityWasAlreadyNamed.cs:33]: Is method name correct?

File: test/StreetNameRegistry.Tests/AggregateTests/WhenCorrectingNisCode/GivenMunicipality.cs [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+            var command = Fixture.Create<CorrectMunicipalityNisCode>()
+                .WithMunicipalityId(_municipalityId);
+
+            Assert(new Scenario()
+                .Given(_streamId,
+                    Fixture.Create<MunicipalityWasImported>())
+                .When(command)
+                .Then(new[]
+                {
+                    new Fact(_streamId, new MunicipalityNisCodeWasChanged(command.MunicipalityId, command.NisCode))
```
Comment (jvandaal) [inline test/StreetNameRegistry.Tests/AggregateTests/WhenCorrectingNisCode/GivenMunicipality.cs:39]: Event is WasChanged, but namespace is WhenCorrecting

Comment (ArneD) [reply test/StreetNameRegistry.Tests/AggregateTests/WhenCorrectingNisCode/GivenMunicipality.cs:39]: Correct, it's irrelevant for the domain to know if it's correction

**Review by jvandaal — APPROVED** [review]

**Pull request merged**